standard-cluster-services/src/main/java/org/terracotta/server/Server.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *
+ *  The contents of this file are subject to the Terracotta Public License Version
+ *  2.0 (the "License"); You may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *
+ *  http://terracotta.org/legal/terracotta-public-license.
+ *
+ *  Software distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
+ *  the specific language governing rights and limitations under the License.
+ *
+ *  The Covered Software is Entity API.
+ *
+ *  The Initial Developer of the Covered Software is
+ *  Terracotta, Inc., a Software AG company
+ *
+ */
+package org.terracotta.server;
+
+import java.util.List;
+import java.util.Properties;
+
+/**
+ *
+ */
+public interface Server {
+  int getServerCount();
+  
+  String[] processArguments();
+
+  void stop(StopAction...modes);
+
+  boolean stopIfPassive(StopAction...modes);
+
+  boolean stopIfActive(StopAction...modes);
```

Comment:
`PlatformService` has the same methods / enum but in/with different classes.
Couldn't we have the same everywhere ?
Or Have `PlatformService` return a `Server` object and provide default implementations in the `PlatformService` interface if we really want backward compat' ?

Reply:
I'd rather leave PatformService as is for backwards compatibility.  The Server global is a necessary evil and I don't want the service to reply on it.